lib/resources/r4/charge_item.yaml

```diff
@@ -100,25 +100,51 @@ fields:
     description: A defined group of bill codes that drives billing behavior.
     url: https://fhir.cerner.com/millennium/r4/financial/general/charge-item#extensions
     binding:
-      description: A defined group of bill codes that drives billing behavior. This binding is used in <extension>bill-code-schedule</extension> extensions.
+      description: A defined group of bill codes that drives billing behavior. This binding is used in <code>bill-code-schedule</code> extensions.
       terminology:
         - display: Bill Code Schedule
           system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/14002
           info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#code-set-14002-bill-code-schedule
 
+  - name: Unit Price/Net Price
+    terminology_name: .extension[x].valueMoney.currency
+    required: 'No'
+    type: Extension
+    description: Currency codes from ISO 4217 (see https://www.iso.org/iso-4217-currency-codes.html).
+    url: https://fhir.cerner.com/millennium/r4/financial/general/charge-item#extensions
+    binding:
+      description: Currency codes from ISO 4217 (see https://www.iso.org/iso-4217-currency-codes.html). This binding is used in <code>net-price</code> and <code>unit-price</code> extensions.
+      terminology:
+        - display: Currencies
+          system: urn:iso:std:iso:4217
+          info_link: http://hl7.org/fhir/r4/valueset-currencies.html
+
   - name: National Drug Product Extension
     terminology_name: extension[x].extension[x].valueCodeableConcept
     required: 'No'
     type: Extension
     description: The national drug code for pharmacy charges.
     url: https://fhir.cerner.com/millennium/r4/financial/general/charge-item#extensions
     binding:
-      description: The national drug code for pharmacy charges. This binding is used in <extension>national-drug-product</extension> extensions.
+      description: The national drug code for pharmacy charges. This binding is used in <code>national-drug-product</code> extensions.
       terminology:
         - display: National Drug Product
           system: http://hl7.org/fhir/sid/ndc
           info_link: http://hl7.org/fhir/sid/ndc
 
+  - name: National Drug Product Extension
```

Comment:
in the name can we add that its for national-drug-product-quantity?

Maybe: National Drug Product Extension - National Drug Product Quantity 
or just 
National Drug Product Quantity

Reply:
Done https://github.com/cerner/fhir.cerner.com/pull/727/commits/8a15adbb97966b3f6d2a674ce02334b504011903#diff-7ab995fcc2135db120dc94b545fa48898edd21d9d220eee5efae95920c369dcbR135